mcache/README.md

```diff
@@ -26,6 +26,14 @@ Follow the instructions below to configure this check for an Agent running on a
         ## url used to connect to the Memcached instance.
         #
         - url: localhost
+
+      # ...
+
+      # logs:
+			#   - type: file
```

Comment:
You've done something weird here, it shows red :S

Reply:
@hithwen fixed it, likely was a copy paste issue